### DISCUSSION TITLE
Fails to compile on Fedora 39

### DISCUSSION THREAD

**Discussion by CorneliusCornbread** [Almamu/linux-wallpaperengine#212] [Q&A]
I've installed all of the libx11 and libxxf86vm and their respective 'devel' packages however, attempting to run `make` yields the following error

```
[  1%] Linking CXX executable linux-wallpaperengine
/usr/bin/ld: cannot find -lX11_Xxf86vm_LIB-NOTFOUND: No such file or directory
collect2: error: ld returned 1 exit status
make[2]: *** [CMakeFiles/linux-wallpaperengine.dir/build.make:1797: linux-wallpaperengine] Error 1
make[1]: *** [CMakeFiles/Makefile2:83: CMakeFiles/linux-wallpaperengine.dir/all] Error 2
make: *** [Makefile:136: all] Error 2
```

I'm completely unable to find why it's referring to libxxf86vm as '-lX11_Xxf86vm_LIB-NOTFOUND' instead of its actual library name. Looking it up it yields absolutely nothing similar to the name given in the error

**Comment by Flashboyv2** [comment]
Is libXxf86vm-devel installed?

**Comment by CorneliusCornbread** [comment]
> Is libXxf86vm-devel installed?

Yes `libXxf86vm-devel` is installed, without it installed it yields the normal, missing `libXxf86vm` error that other people on StackOverflow report when they're missing it.

**Comment by AngelSherry** [comment]
Well, here are my requires of RPM SPEC file when building on Nobara/Fedora 39, which is building successfully on [my COPR](https://copr.fedorainfracloud.org/coprs/yohane-shiro/linux-wallpaperengine/). You may refer to these requires.

```
BuildRequires: gcc-c++ cmake libXrandr-devel freeimage-devel libXinerama-devel libXcursor-devel libXi-devel mesa-libGL-devel glew-devel freeglut-devel SDL2-devel lz4-devel ffmpeg-free-devel libXxf86vm-devel glm-devel glfw-devel mpv-devel pulseaudio-libs-devel

Requires: glfw mpv mpv-libs pulseaudio-libs
```
